ibm-cloud-architecture/Learning-Kabanero-101
# Updated docs for OCP 4.2

**Issue**
Docs may or may not have been out of date for OCP 4.2.

**What I did**
Updated the e2e docs for nodejs, springboot and microprofile microservice creation with appsody.

**How to test**
Pull and build and check and see if docs are sufficient and wording is correct. Also ensure new images are relevant.
